docs/release-notes/v0.27.0.md

```diff
@@ -0,0 +1,126 @@
+## Announcing Radius v0.26.0
+
+Today we're happy to announce the release of Radius v0.27.0. Check out the [highlights](#highlights) below, along with the [full changelog](#full-changelog) for more details.
+
+We would like to extend our thanks to all the [new](#new-contributors) and existing contributors who helped make this release possible!
+
+## Intro to Radius
+
+If you're new to Radius, check out our website, [radapp.io](https://radapp.io), for more information. Also visit our [getting started guide](https://docs.radapp.io/getting-started/) to learn how to install Radius and create your first app.
+
+## Highlights
+
+### Recipe Template Information in Resource Status
+Recipe's template information is now available in the resource status. This information is useful to understand the template used to create the resource.
+
+### Improved Error Messages
+Error messages formatting have been improved to easy to read and understand.
```

Comment:
```suggestion
### Mock environments

Radius now supports mock environments, where application deployments do not result in any running containers or deployed infrastructure. These environments are useful when doing dry-run/test deployments or to interact with the Radius application graph on an application template before it's fully deployed. Learn more in the [environment docs](https://docs.radapp.io/reference/resource-schema/core-schema/environment-schema/).

### Application graph API

A new app graph API allows you to now get all resources and connections within an application via a single API call, instead of building the graph manually via multiple APIs. This API unlocks custom visualizations and experiences on top of Radius. Learn more in the [Radius API docs](https://docs.radapp.io/reference/api/applications.core/applications/#tag/Applications/operation/Applications_GetGraph).

### Kubernetes annotation-based Radius apps

Want to add Radius to an existing Kubernetes application but don't want to rewrite it in Bicep?  Radius now supports annotations and CRDs that allow you to add your app to the Radius application graph, deploy Recipes, and easily configure connections. Learn more in the [new tutorial](https://docs.radapp.io/tutorials/helm/).

### eShop on Radius with Recipes

The [eShop on containers] reference app is now Recipe-enabled, providing a single application definition that works on local-dev, Azure, and AWS environments. Learn more about eShop [in the docs](https://docs.radapp.io/tutorials/eshop/).

### Bug fixes and performance improvements

We've fixed a bunch of installation bugs and experience issues based on the amazing user feedback that you have all been submitting. Please make sure to continue to [log Issues](https://github.com/radius-project/radius/issues/new/choose) and report issues [in Discord](https://aka.ms/Radius/Discord).
```